torch/utils/flop_counter.py

```diff
@@ -89,6 +89,22 @@ def baddbmm_flop(self_shape, a_shape, b_shape, out_shape=None, **kwargs) -> int:
     # Inputs contains the shapes of three tensors.
     return bmm_flop(a_shape, b_shape)
 
+@register_flop_formula(aten._scaled_mm)
+def scaled_mm_flop(
+    a_shape,
+    b_shape,
+    scale_a_shape,
+    scale_b_shape,
+    bias_shape=None,
+    scale_result_shape=None,
+    out_dtype=None,
+    use_fast_accum=False,
+    out_shape=None,
+    **kwargs,
+) -> int:
+    """Count flops for _scaled_mm."""
+    return mm_flop(a_shape, b_shape)
```

Comment:
I'm guessing there is logic elsewhere in the stack to adjust the cost based on bytes per element?

Reply:
Nope, there isn't, but I flagged this a few days ago in https://github.com/pytorch/pytorch/issues/144470.

Note, however, that it wouldn't fully solve the problem, because different kernels have different levels of efficiency (e.g., matmuls might be 80% optimal, but Flash might only be 50%). So when using FLOP counts to estimate kernel runtime we should also take that into account. Or we just resort to profiling, but that has issues too (divergence between ranks).